src/Foundation/Enums.cs

```diff
@@ -125,4 +125,122 @@ public enum NSUrlSessionMultipathServiceType : long {
 		Interactive = 2,
 		Aggregate = 3,
 	}
+
+	public enum NSLinguisticTagScheme {
+		[Field ("NSLinguisticTagSchemeTokenType")]
+		Token,
+
+		[Field ("NSLinguisticTagSchemeLexicalClass")]
+		LexicalClass,
+
+		[Field ("NSLinguisticTagSchemeNameType")]
+		Name,
+
+		[Field ("NSLinguisticTagSchemeNameTypeOrLexicalClass")]
+		NameOrLexicalClass,
+
+		[Field ("NSLinguisticTagSchemeLemma")]
+		Lemma,
+
+		[Field ("NSLinguisticTagSchemeLanguage")]
+		Language,
+
+		[Field ("NSLinguisticTagSchemeScript")]
+		Script,
+	}
+
+	public enum NSLinguisticTagUnit {
```

Comment:
both swift and ObjC uses `NSLinguisticTag` so let's do the same

https://developer.apple.com/documentation/foundation/nslinguistictag/1418153-word

Reply:
Issue, we have a static interface that exposes the values, I will add a deprecation, will state that the enum should be used. Use the TagUnit name in the enum (with a XAMCORE 4 check to rename accordingly for the new API change).